packages/financial-templates-lib/src/logger/PagerDutyV2Transport.ts

```diff
@@ -1,13 +1,20 @@
 // This transport enables winston logging to send messages to pager duty v2 api.
 import Transport from "winston-transport";
 import { event } from "@pagerduty/pdjs";
+import * as ss from "superstruct";
 
 type TransportOptions = ConstructorParameters<typeof Transport>[0];
 export type Severity = "critical" | "error" | "warning" | "info";
 
-interface Config {
-  integrationKey: string;
-  customServices?: { [key: string]: string };
+const Config = ss.object({
+  integrationKey: ss.string(),
+  customServices: ss.optional(ss.record(ss.string(), ss.string())),
```

Comment:
Is there any way to make these nested mappings more readable?

I really like superstruct, but the actual expected shape is hard to understand from this. If we can't make it more readable, could we add a comment showing the expected shape?

Reply:
ill see waht i can do